modules/arrow/src/geoarrow/convert-geojson-feature-to-arrow.ts

```diff
@@ -0,0 +1,375 @@
+import {
+  Float64 as ArrowFloat,
```

Comment:
Since this renaming tends to get excessive, in other files I have been breaking my own rule (not to use wildcard imports) and used `import * as arrow` and then referenced as `arrow.Float`. I think this makes the code quite clear.